ios/MusicControlManager.m

```diff
@@ -370,8 +370,9 @@ - (void)audioInterrupted:(NSNotification *)notification {
     }
     NSInteger interruptionType = [notification.userInfo[AVAudioSessionInterruptionTypeKey] integerValue];
     NSInteger interruptionOption = [notification.userInfo[AVAudioSessionInterruptionOptionKey] integerValue];
-
-    if (interruptionType == AVAudioSessionInterruptionTypeBegan) {
+    bool delayedSuspendedNotification = (@available(iOS 10.0, *)) && [notification.userInfo[AVAudioSessionInterruptionWasSuspendedKey] integerValue] == 1;
```

Comment:
do you think boolValue can work here so that we don't need to check with `1`?

Reply:
Absolutely, I'm a bit new to Objective-C.  Switched this over to use `boolValue`